Tasks/Common/nuget-task-common/NuGetToolRunner2.ts

```diff
@@ -98,14 +98,36 @@ export class NuGetToolRunner2 extends ToolRunner {
     public execSync(options?: IExecOptions): IExecSyncResult {
         options = options || <IExecOptions>{};
         options.env = prepareNuGetExeEnvironment(options.env || process.env, this.settings, this.authInfo);
-        return super.execSync(options);
+        let execResult = super.execSync(options);
+        if (execResult.code !== 0) {
+            this._logExecResults(execResult.code, execResult.stderr);
+        }
+        return execResult;
     }
 
     public exec(options?: IExecOptions): Q.Promise<number> {
         options = options || <IExecOptions>{};
         options.env = prepareNuGetExeEnvironment(options.env || process.env, this.settings, this.authInfo);
         return super.exec(options);
     }
+
+
+    private _logExecResults(exitCode: number, stderr: string){
+        try{
+            console.log("##vso[telemetry.publish area=Packaging;feature=NuGetCommand]%s",
```

Comment:
The agent checks (if needed) are not being made in any of the tool runner libs.

Reply:
Fixed

Reply:
I see your comment, but I don't see the agent checks. Forgot to push?